src/main/scala/com/gu/configraun/Configraun.scala

```diff
@@ -1,17 +1,29 @@
 package com.gu.configraun
 
 import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagement
-import com.gu.configraun.Errors.ConfigraunError
-import com.gu.configraun.aws.GetParametersByPathResultProcessor
-import com.gu.configraun.models._
+import com.gu.configraun.Errors.{ConfigException, ConfigraunError}
+import com.gu.configraun.aws.{AwsInstanceTags, GetParametersByPathResultProcessor}
+import com.gu.configraun.models.{Configuration, PROD, Stage}
 
 
 object Configraun {
 
   def loadConfig(stack: String, app: String, stage: Stage)(implicit client: AWSSimpleSystemsManagement): Either[ConfigraunError, Configuration] = loadRemoteConfiguration(stack, app, stage)
+  def loadConfig(stack: String, app: String, stage: String)(implicit client: AWSSimpleSystemsManagement): Either[ConfigraunError, Configuration] = loadRemoteConfiguration(stack, app, stage)
```

Comment:
Given we're passing stack, app, stage around now it might be worth building a context object for this.